pytube/query.py

```diff
@@ -162,9 +162,23 @@ def order_by(self, attribute_name):
         :param str attribute_name:
             The name of the attribute to sort by.
         """
+        integer_attr_repr = {}
+        for stream in self.fmt_streams:
+            attr = getattr(stream, attribute_name)
+            if attr is None:
+                break
+            num = [char for char in attr if char.isdigit()]
+            integer_attr_repr[attr] = int(''.join(num)) if num else None
+
+        # if every attribute has an integer representation
+        if all(integer_attr_repr.values()) and integer_attr_repr:
```

Comment:
Shouldn't this be `integer_attr_repr and all(integer_attr_repr.values())`?

Reply:
just curious, is short circuiting the only reason? readablilty? either way I'll make the change